app/src/main/java/com/hover/stax/inapp_banner/BannerViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.hover.stax.inapp_banner
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.viewModelScope
+import com.hover.stax.database.DatabaseRepo
+import kotlinx.coroutines.launch
+
+class BannerViewModel(application: Application, repo: DatabaseRepo) : AndroidViewModel(application) {
+    private val qualifiedBannerLiveData: MutableLiveData<Banner> = MutableLiveData<Banner>()
+    private val bannerUtils = BannerUtils(getApplication())
```

Comment:
Use the injected application instance in the constructor.

Reply:
Thanks, done this